include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -6369,18 +6369,32 @@
           "extensions" : [ "SPV_NV_fragment_shader_barycentric" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "FragSizeEXT",
```

Comment:
Putting the EXT names first will cause the disassembler to prefer the EXT names when outputting.  That's a good choice.  The only impact would be needing an update to SPIRV-Tools tests (if any have been made for the corresponding NV extension.)